compiler/src/typed/types.re

```diff
@@ -141,6 +141,7 @@ type value_description = {
   val_type: type_expr,
   val_repr,
   val_kind: value_kind,
+  val_internalpath: Path.t,
```

Comment:
Does this remove the need to use `Env.find_value` in #1241 ??

Reply:
Nah, that's more about locations.